frontend/src/app/app-body/app-body.tsx

```diff
@@ -20,15 +22,28 @@ export class AppBody extends React.Component<IProps> {
   public render() {
     return (
       <Row gutter={16}>
-        <Col span={12}>
+        <Col span={8}>
           <div
             style={{ color: this.props.testStore.colourActive ? 'red' : '' }}
           >
-            Toggle my colour!
+            Toggle the colour of this text!
           </div>
         </Col>
 
-        <Col span={12}>
+        <Col span={8}>
```

Comment:
why 8?

Reply:
So the width of a row is 24 defined by ant design, we have 3 components in the row so this is set to 8